### PR TITLE
Updated Automapper to 6.2.2

### PR DESCRIPTION
**Pull request by rileymckenna**
## Description

* Updated Automapper 6.2.0 -> 6.2.2 
* Bug fix in Get-AzPeeringLocation 

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Peering/Peering/Az.Peering.psd1 [hunk 1]
```diff
@@ -53,7 +53,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.5.0'; })
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.5.1'; })
```
Comment (cormacpayne) [inline src/Peering/Peering/Az.Peering.psd1]: @rileymckenna this should be bumped up to 1.5.2

File: src/Peering/Peering/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+## Version 0.1.1
+* Updated Automapper 6.2.0 -> 6.2.2 
+* Bug fix in Get-AzPeeringLocation
```
Comment (cormacpayne) [inline src/Peering/Peering/ChangeLog.md]: @rileymckenna what was the bug fix to this cmdlet?

File: src/Peering/Peering/ChangeLog.md [hunk 3]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+## Version 0.1.1
```
Comment (cormacpayne) [inline src/Peering/Peering/ChangeLog.md]: @rileymckenna you can remove the `## Version 0.1.1` header and keep everything under `## Upcoming Release` -- we will do the versioning

File: src/Peering/Peering/Az.Peering.psd1 [hunk 4]
```diff
@@ -121,7 +121,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Preview Release for Microsoft Peering Service'
+        ReleaseNotes = '* Updated AutoMapper to 6.2.2'
```
Comment (cormacpayne) [inline src/Peering/Peering/Az.Peering.psd1]: @rileymckenna you can revert the change made here -- we'll do this as a part of versioning

File: src/Peering/Peering/Az.Peering.psd1 (hunk 1 above)
Comment (rileymckenna) [reply src/Peering/Peering/Az.Peering.psd1]: @cormacpayne should i increment the version of the package to 0.1.1?

File: src/Peering/Peering/ChangeLog.md (hunk 2 above)
Comment (rileymckenna) [reply src/Peering/Peering/ChangeLog.md]: so my changes are definitely in my code, i must have slipped them in the first PR before it got merged.

**Comment by cormacpayne** [comment]
Closing in favor of PR https://github.com/Azure/azure-powershell/pull/9216
